app/logging.js

```diff
@@ -117,8 +117,8 @@ function eliminateOutOfDateFiles(logPath, date) {
       const file = {
         path: target,
         start: isLineAfterDate(start, date),
-        end: isLineAfterDate(end[end.length - 1], date)
-          || isLineAfterDate(end[end.length - 2], date),
+        end: isLineAfterDate(end[end.length - 1], date) ||
+             isLineAfterDate(end[end.length - 2], date),
```

Comment:
Auto-fixes from the new ESLint rule.

Reply:
Do you like this formatting? This is our opportunity to choose the rules we want.

Reply:
Per discussion, enabled this globally as it caused less churn.